package.json

```diff
@@ -36,8 +36,8 @@
   ],
   "scripts": {
     "clean": "rimraf lib dist es coverage",
-    "format": "prettier --write \"{src,test}/**/*.js\"",
-    "format:check": "prettier --list-different \"{src,test}/**/*.js\"",
+    "format": "prettier --write \"{src,test}/**/*.js\" \"{src,test}/**/*.ts\"",
+    "format:check": "prettier --list-different \"{src,test}/**/*.js\" \"{src,test}/**/*.ts\"",
```

Comment:
You should be using the same pattern approach as for the folders i.e `{src,test}/**/*.{js,ts}`

Reply:
done.